charts/add-ons/grafana/templates/grafana-rbac.yaml

```diff
@@ -11,3 +11,7 @@ metadata:
   labels:
     {{.Values.global.controllerComponentLabel}}: grafana
     {{.Values.global.controllerNamespaceLabel}}: {{.Values.global.namespace}}
+{{- if .Values.global.imagePullSecrets }}
```

Comment:
because `partials` is also a sub-chart for add-ons, `{{- include "partials.image-pull-secrets" . }}` should work directly

Reply:
I tried this, but were failing tests so on a later commit i  introduced this.

Reply:
Just using `{{- include "partials.image-pull-secrets" . }}` works for me... Can you push it just with that, and so we can see what's the actual failure in CI?

Reply:
@apleb, @Pothulapati  done